scripts/drop_issuer_nf_tokens.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash 
+#Drops issuer_nf_tokens table and set latest ledger index to the one prior to XLS-20 amendment
+
+XLS20_PRIOR_LEDGER_INDEX=75443456
+
+SEQUENCE_1=`cqlsh -e "SELECT sequence FROM clio.ledger_range WHERE is_latest=true;"`
```

Comment:
it's preferable in bash to use this syntax instead of backticks:

```
SEQUENCE_1=$(cqlsh -e "SELECT sequence FROM clio.ledger_range WHERE is_latest=true;")
```

i don't remember why to be honest lol. i think it has to do with character escapes